java/trino/src/main/java/sleeper/trino/remotesleeperconnection/SleeperConnectionAsTrino.java

```diff
@@ -195,6 +195,7 @@ public Stream<List<Object>> streamPartitionStatusRows(SchemaTableName schemaTabl
         List<Partition> allPartitions = this.sleeperRawAwsConnection.getSleeperTablePartitionStructure(
                 schemaTableName.getTableName(),
                 Instant.now()).getAllPartitions();
+        Schema schema = this.sleeperRawAwsConnection.getSleeperSchema(schemaTableName.getTableName());
```

Comment:
The use of this here is unnecessary, see https://github.com/gchq/sleeper/blob/develop/docs/development/conventions.md#use-of-this.

Reply:
Done